### PR TITLE
Add Azure Managed Identities as adopt

### PR DESCRIPTION
**Pull request by Krusty93**
- [X] I've added a short description of the entry that
      clarifies what it does and why it's useful
- [X] I've added links to the entry's documentation and/or
      website
- [X] My frontmatter is correct, complete and follows the
      format specified in the [README](../README.md)
- [x] In case the entry is in the `adopt` category: I've
      added at least one reference to showcase the entry usage
      among our organization
- [x] I've described at least one use case for the entry

### CODE REVIEW

File: radar/platforms-and-services/azure-managed-identity.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: "Azure Managed Identity"
+ring: adopt
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, azure, security]
+---
+
+[Azure Managed Identities](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview) simplify identity management for applications and services running in the cloud. They provide a secure and seamless way for applications to access Azure resources without the need for explicit credentials. Managed Identities are tightly integrated with [Entra ID](https://learn.microsoft.com/en-us/entra/fundamentals/whatis) and eliminate the need for developers to manage and rotate authentication keys manually. This helps enhance security by reducing the risk associated with managing and storing sensitive credentials. Azure Managed Identities come in two types: System-assigned, which is tied to an Azure resource, and User-assigned, which can be associated with multiple resources.
```
Comment (gunzip) [inline radar/platforms-and-services/azure-managed-identity.md:8]: if we don't plan to put Entra ID in our radar I'd remove all references to it.

**Review by gunzip — CHANGES_REQUESTED** [review]
can you add a list of pros against connection strings?

File: radar/platforms-and-services/azure-managed-identity.md (hunk 1 above)
Comment (Krusty93) [reply radar/platforms-and-services/azure-managed-identity.md:8]: We definitely need to add entra id soon

Comment (gunzip) [reply radar/platforms-and-services/azure-managed-identity.md:8]: I suggest to add it in 'assess'.

Comment (Krusty93) [reply radar/platforms-and-services/azure-managed-identity.md:8]: Why? We use it actively (user management, service connection, etc.)

File: radar/platforms-and-services/azure-managed-identity.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Managed identities are preferred over connection strings too. Connection strings have a less role granular approach and make difficult to understand who and what are using them. Connection strings must be treated as secrets and they are subject to leaks if not properly stored. Having a key leak forces to keys rotation, and all applications would need to be properly updated creating a trickle-down effect.
+
+Use system-assigned identities for any service that needs to communicate with others, such as an [App Service](https://azure.microsoft.com/en-us/products/app-service) that wants to read data from a [Storage Account](https://learn.microsoft.com/en-us/azure/storage/common/storage-account-overview).
+
+User-assigned identities are not bond to a specific resource in partiular and they have their own lifecycle. For this reason, they are used mainly to federate a subscription with an external services such as GitHub or Azure DevOps.
```
Comment (gunzip) [inline radar/platforms-and-services/azure-managed-identity.md:18]:
```suggestion
User-assigned identities are not bound to a specific resource in particular and they have their own lifecycle. For this reason, they are primarily used to federate a subscription with external services such as GitHub or Azure DevOps

File: radar/platforms-and-services/azure-managed-identity.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+### Use cases
+
+Managed Identities have a wide usage in our organization. They are the preferred way to authenticate services between each other due to the passwordless approach and the role assignment granularity. Creating a managed identity doesn't require any Entra ID role: RBAC built-in roles are sufficient. This is a strong point to prefer managed identities over service principals.
+
+Managed identities are preferred over connection strings too. Connection strings have a less role granular approach and make difficult to understand who and what are using them. Connection strings must be treated as secrets and they are subject to leaks if not properly stored. Having a key leak forces to keys rotation, and all applications would need to be properly updated creating a trickle-down effect.
```
Comment (gunzip) [inline radar/platforms-and-services/azure-managed-identity.md:14]:
```suggestion
Managed identities are often preferred over connection strings. Connection strings are less granular in terms of role specification and make it difficult to identify who and what are using them. These strings should be treated as secrets, as they are at risk of being leaked if not securely stored. A key leak necessitates key rotation, creating a trickle-down effect where all related applications need to be updated accordingly.
```

**Review by gunzip — APPROVED** [review]
very good

**Comment by gunzip** [comment]
@iwoak lgtm

**Review by uolter — APPROVED** [review]

**Pull request merged**